src/Microsoft.ML.Core/Data/Schema.cs

```diff
@@ -72,6 +74,11 @@ public Column GetColumnOrNull(string name)
             return null;
         }
 
+        public override string ToString()
```

Comment:
Do we need these `ToString` overrides on types that have `DebuggerTypeProxy` attributes?  Here and `Metadata` below.

Reply:
Yes. The `ToString` is displayed when the entry is not expanded, and

---
In reply to: [230909215](https://github.com/dotnet/machinelearning/pull/1518#discussion_r230909215) [](ancestors = 230909215)